tools/snp.sh

```diff
@@ -1187,6 +1264,11 @@ main() {
   if ! $UPM; then
     SETUP_WORKING_DIR="${SETUP_WORKING_DIR}/non-upm"
   fi
+  
+  # Set SETUP_WORKING_DIR for non-upm
+  #if [ "$SVSM" = true ]; then
+   # SVSM_SETUP_WORKING_DIR="${SETUP_WORKING_DIR}/svsm"
+  #fi
```

Comment:
Do we intend to support non-UPM?

Reply:
Idk if there are svsm kernels that support non-upm